src/claimtrie.cpp

```diff
@@ -2,10 +2,24 @@
 #include "coins.h"
 #include "hash.h"
 
-#include <boost/scoped_ptr.hpp>
 #include <iostream>
 #include <algorithm>
 
+// leveldb keys
+#define HASH_BLOCK 'h'
+#define CURRENT_HEIGHT 't'
+#define TRIE_NODE 'n'
+#define CLAIM_BY_ID 'i'
+#define CLAIM_QUEUE_ROW 'r'
+#define CLAIM_QUEUE_NAME_ROW 'm'
+#define EXP_QUEUE_ROW 'e'
+#define SUPPORT 's'
+#define SUPPORT_QUEUE_ROW 'u'
+#define SUPPORT_QUEUE_NAME_ROW 'p'
+#define SUPPORT_EXP_QUEUE_ROW 'x'
+
```

Comment:
I think these can go in claimtriedb.h

Reply:
I've give a try to implement it on elegant way. Generic in claimtriedb as
```
template<typename K, typename V>
bool getQueue(const K &key, V &value) const
{
    auto hash = typeid(std::map<K, V>).hash_code();
    auto it = map_hashes.find(hash);
    if (it == map_hashes.end()) return false;
    auto map = reinterpret_cast<std::map<K, V>>(it->second);
    auto i = map.find(key);
    if (i != map.end()) { value = i->second; return true; }
    return false;
}
```
I mean it can be used a hash to store generic queues so all definitions will gone away, but we should provide db migration, right?